libkbfs/keybase_service_base.go

```diff
@@ -437,19 +445,68 @@ func (k *KeybaseServiceBase) LoadUserPlusKeys(ctx context.Context,
 	return k.processUserPlusKeys(res)
 }
 
+var allowedLoadTeamRoles = map[keybase1.TeamRole]bool{
+	keybase1.TeamRole_NONE:   true,
```

Comment:
I see; thanks. I guess I should've asked what role `TeamRole_NONE` represents. But you just answered that it's used as a zero value when the UID is not set.